src/components/input/submit/index.tsx

```diff
@@ -29,7 +29,11 @@ export class SmoothlyInputSubmit implements ComponentWillLoad {
 					this.display = !p.readonly
 					this.disabled =
 						!this.delete &&
-						(p.readonly ? true : Object.values(p.value).filter(val => val).length < 1 ? true : !p.changed)
+						(p.readonly
+							? true
+							: Object.values(p.value).filter(val => val).length < 1
+							? true
+							: !p.changed || this.disabled)
```

Comment:
`p.readonly || Object.values(p.value).filter(val => val).length < 1 || !p.changed || this.disabled`

Reply:
There is no need to do 
`
a == 1 ? true : false
`

Reply:
absolutely